srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/Pages/SoapLogs.php

```diff
@@ -0,0 +1,146 @@
+                            <input type="text" class="form-control form-control-sm" id="order_id" name="order_id" placeholder="Order ID" value="<?php echo $data['order_id']; ?>">
+                        </div>
+
+                        <div class="col">
+                            <label class="visually-hidden" for="items">Items</label>
+                            <select class="form-select" name="items" id="items">
+                                <option value="10">10</option>
+                                <option value="25" <?php echo $data['items'] == '25' ? 'selected' : ''; ?>>25</option>
+                                <option value="50" <?php echo $data['items'] == '50' ? 'selected' : ''; ?>>50</option>
+                                <option value="100" <?php echo $data['items'] == '100' ? 'selected' : ''; ?>>100</option>
+                            </select>
+                        </div>
+
+                        <div class="col">
+                            <button type="submit" class="btn btn-primary btn-sm">Search</button>
+                        </div>
+
+                        <div class="col ms-auto">
+                            <nav aria-label="Table pagination">
+                                <ul class="pagination pagination-sm">
+                                    <?php
+                                        $maxPagination = 3;
+                                        $diff = $data["max_page"] - $data["pos"];
+                                        $start = $data["pos"] - $maxPagination > 0 && $diff >= 0 ? $data["pos"] - $maxPagination : 1;
+                                        $end = $data["pos"] + $maxPagination < $data["max_page"] && $diff > 0 ? $data["pos"] + $maxPagination : $data["max_page"];
+
+                                        $link = menu_page_url(ACORE_SLUG . '-soap-logs', false);
+                                        if ($data["username"]) {
+                                            $link .= "&username" . $data["username"];
+                                        }
+                                        if ($data["order_id"]) {
+                                            $link .= "&order_id" . $data["order_id"];
+                                        }
+                                        ?>
+
+                                        <li class="page-item disabled">
+                                        <?php if ($data["count"] > 0): ?>
+                                        <a class="page-link">Results <?php echo ($data["pos"] - 1) * $data["items"] + 1; ?> to <?php echo $data["pos"] != $data["max_page"] ? $data["pos"] * $data["items"] : $data["count"]; ?> from <?php echo $data["count"]; ?> </a>
+                                        <?php else: ?>
+                                            <a class="page-link">No results </a>
+                                        <?php endif; ?>
```

Comment:
I never used this `if :` and `else :` with `endif`, I usually leave an open bracket `if () { ?>` ... `<?php } else { ?>`, I don't know what is the best approach